### PR TITLE
Test NSM Suites: Update cloudtest tool, split executions #1

### PR DESCRIPTION
**Pull request by denis-tingajkin**
## Motivation and Context

Update / split executions from cloudtest config

Changes from this PR need to https://github.com/networkservicemesh/networkservicemesh/issues/2070

## How Has This Been Tested?

- [X] Covered by existing integration testing
- [ ] Added integration testing to cover
- [X] Tested locally
- [ ] Have not tested

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -160,8 +160,11 @@ jobs:
               export STORE_LOGS_IN_ANY_CASES=true
             fi
             export STORE_JAEGER_TRACES=true
-            pushd test
-            go get github.com/networkservicemesh/cloudtest
+            git clone https://github.com/networkservicemesh/cloudtest.git
```
Comment (denis-tingajkin) [inline .circleci/config.yml]: TODO: revert these changes when PR https://github.com/networkservicemesh/cloudtest/pull/3 will be merged

Comment (haiodo) [reply .circleci/config.yml]: Nice idea  :-)
